modules/network-flow-matrix.adoc

```diff
@@ -6,26 +6,91 @@
-The network information in the matrix is accurate for both bare-metal and cloud environments.
-Use the information in the network flow matrix to help you manage ingress traffic.
-You can restrict ingress traffic to essential flows to improve network security.
+The following network flow matrixes describe the ingress flows to {product-title} services for the following environments:
 
-To view or download the raw CSV content, see link:https://raw.githubusercontent.com/openshift/openshift-docs/main/snippets/network-flow-matrix.csv[this resource].
+* {product-title} on bare metal 
+* {sno} on bare metal
+* {product-title} on {aws-first}
+* {sno} on {aws-short}
 
-Additionally, consider the following dynamic port ranges when managing ingress traffic:
+Use the information in the appropriate network flow matrix to help you manage ingress traffic for your specific environment. You can restrict ingress traffic to essential flows to improve network security.
+
+Additionally, consider the following dynamic port ranges when managing ingress traffic for both bare metal and cloud environments:
 
 * `9000-9999`: Host level services
 * `30000-32767`: Kubernetes node ports
 * `49152-65535`: Dynamic or private ports
 
+To view or download the complete raw CSV content for an environment, see the following resources:
+
+//link:https://raw.githubusercontent.com/openshift/openshift-docs/main/snippets/network-flow-matrix.csv[this resource].
+
+* link:https://raw.githubusercontent.com/rohennes/openshift-docs/TELCODOCS-2015/snippets/communication-matrix-bm.csv[{product-title} on bare metal]
+
+* link:https://raw.githubusercontent.com/rohennes/openshift-docs/TELCODOCS-2015/snippets/communication-matrix-bm-sno.csv[{sno} on bare metal]
+
+* link:https://raw.githubusercontent.com/rohennes/openshift-docs/TELCODOCS-2015/snippets/communication-matrix-aws.csv[{product-title} on {aws-short}]
+
+* link:https://raw.githubusercontent.com/rohennes/openshift-docs/TELCODOCS-2015/snippets/communication-matrix-aws-sno.csv[{sno} on {aws-short}]
+
 [NOTE]
 ====
-The network flow matrix describes ingress traffic flows for a base {product-title} installation. It does not describe network flows for additional components, such as optional Operators available from the Red Hat Marketplace. The matrix does not apply for Hosted-Control-Plane, MicroShift, or standalone clusters.
+The network flow matrixes describe ingress traffic flows for a base {product-title} or {sno} installation. It does not describe network flows for additional components, such as optional Operators available from the Red Hat Marketplace. The matrixes do not apply for Hosted-Control-Plane, MicroShift, or standalone clusters.
 ====
 
-.Network flow matrix
+[id="network-flow-matrix-common_{context}"]
+== Network flow matrix base flows
+
+The following matrixes describe the ingress flows to {product-title} services that are common to the following environments:
```

Comment:
I'm not sure about mentioning the list of the platforms, I was thinking this should talk about standard OCP and mention here for SNO reference control plane list only.
Should we also mention that additional flows per platform can be found next?

Reply:
I removed the list of platforms.

I added a note about SNO.

I don't know if we need to say additional flows are later in the content, I've updated the headings so I think it should be apparent.